pkg/shared/families/malware/clam/clam.go

```diff
@@ -112,7 +130,7 @@ func (s *Scanner) Run(sourceType utils.SourceType, userInput string) error {
 }
 
 func New(c job_manager.IsConfig, logger *log.Entry, resultChan chan job_manager.Result) job_manager.Job {
-	conf := c.(*common.ScannersConfig) // nolint:forcetypeassert
+	conf := c.(common.ScannersConfig) // nolint:forcetypeassert
```

Comment:
Why the change here is needed? All other families are casting to a pointer.

Reply:
I need to root cause this some more, but I was always getting a nil pointer here which was causing a panic at L137 because it was trying to access the "Clam" field. I'll do some more digging

Reply:
I reverted this and it no longer gave me a nil pointer issue... I think this might be caused by not having the malware section in the families.yaml, we need to handle defaults a bit better.